src/shared/api/io.test.ts

```diff
@@ -1,28 +1,23 @@
 import { fail } from 'assert'
```

Comment:
relly.. you want to throw away Asgardex's ledger tests.. um no. Im this close to booting you from submitting another pr

Reply:
Added back